plone/app/portlets/portlets/actions.pt

```diff
@@ -4,15 +4,15 @@
       tal:omit-tag=""
       i18n:domain="plone">
 
-<section class="portlet portletActions" role="menu"
+<div class="portlet portletActions"
```

Comment:
I would imagine an `aside` should have more to do with the main content. A list of related items could be an `aside`. But the actions listed here are the same across all pages: they belong to the site, not really the page.